packages/components/ng-ovh-line-diagnostics/src/translations/Messages_de_DE.json

```diff
@@ -113,7 +113,7 @@
   "tools_lineDiagnostics_solution_proposal_step_request_form_question_followBySms_title": "Souhaitez-vous suivre le déroulement des opérations par SMS ?",
   "tools_lineDiagnostics_solution_proposal_step_request_form_question_ovhTicket_title": "Si un ticket est actuellement ouvert pour ce dysfonctionnement, merci de nous en indiquer le numéro.",
   "tools_lineDiagnostics_solution_proposal_step_request_form_question_comment_title": "Précisez ici toutes les informations nécessaires au bon rétablissement de votre connexion Internet, ainsi que vos disponibilités. (Entre 5 et 200 caractères)",
-  "tools_lineDiagnostics_solution_proposal_step_request_form_question_conditionsAccepted_title": "J'ai pris connaissance du fait qu'une facturation de 150 euros HT sera appliquée si le dysfonctionnement relève de ma responsabilité. J'ai bien suivi toutes les manipulations demandées jusqu'à présent.",
+  "tools_lineDiagnostics_solution_proposal_step_request_form_question_conditionsAccepted_title": "J'ai pris connaissance du fait qu'une facturation de 199 euros HT sera appliquée si le dysfonctionnement relève de ma responsabilité. J'ai bien suivi toutes les manipulations demandées jusqu'à présent.",
```

Comment:
maybe it would be more maintainable to add a variable like {{ amount }} and sets the value outside with translate-values, what do you think?